src/backend/marsha/core/tests/test_api_liveregistration.py

```diff
@@ -2125,7 +2241,7 @@ def test_list_liveregistration_public_token(
     def test_api_liveregistration_list_throttling(self):
         """Throttling should prevent more than three requests per minute."""
         # first 3 requests shouldn't be throttled
-        for _ in range(3):
+        for _i in range(3):
```

Comment:
Does the linter warn you to make this change ?

Reply:
yes but it's only due to the import _ for the lasy translations, it brings conflict

Reply:
:+1: